src/panels/energy/strategies/energy-strategy.ts

```diff
@@ -141,6 +141,11 @@ export class EnergyStrategy {
         view_layout: { position: "sidebar" },
         collection_key: "energy_dashboard",
       });
+      view.cards!.push({
```

Comment:
This could also be conditioned by `hasSolar && hasGrid`.
This would have one minor drawback. The calculated self-sufficiency value would be off if someone returned 'unmeasured' energy to the grid, as we assume it has been used up in the home.